tests/functional-tests/README.md

```diff
@@ -16,62 +13,60 @@ This project contains an automation framework to exercise the IBM Spectrum Scale
 
 Install Python (3.7.4 or higher) and below mentioned pip modules:
 
-  ``` 
-        python3.7 -m pip install kubernetes
-        python3.7 -m pip install pytest
-        python3.7 -m pip install pytest-html
-        python3.7 -m pip install jsmin
-  ```
-       
+``` 
+python3.7 -m pip install kubernetes
+python3.7 -m pip install pytest
+python3.7 -m pip install pytest-html
+python3.7 -m pip install jsmin
+```
```

Comment:
Not really a required change, just just FYI... it may be useful to start creating `requirements.txt` files and pass those into pip... this way this file could be used by dev/test teams and if requirements, updating requirements file would automatically install the right pre-reqs for the testing suite.  Instead of itemizing them one by one in a README....